tools/generate-gemm-test.py

```diff
@@ -236,93 +236,52 @@ def split_ukernel_name(name):
         , test_func, isa_check)
         .loop_n(1, nr)
         .loop_m(1, mr));
-  if (k_block > 1) {
```

Comment:
FYI The diff is bit confusing but all that actually changed here is that I wrapped the k_lt and k_gt tests in an $if not blockwise.